banzai/main.py

```diff
@@ -85,7 +85,7 @@ def get_preview_stages_todo(image_suffix):
                                  extra_stages=[dark.DarkNormalizer, dark.DarkComparer])
     elif image_suffix == 'f00.fits':
         stages = get_stages_todo(last_stage=dark.DarkSubtractor,
-                                 extra_stages=[flats.FlatNormalizer, flats.FlatComparer])
+                                 extra_stages=[flats.FlatNormalizer, qc.PatternNoiseDetector, flats.FlatComparer])
```

Comment:
Why do we want to run this so late in the process? Why not run it earlier?

Reply:
I think we decided to run the pattern noise checker on the reduced data, just to minimize occurrences of FFT artifacts due to e.g. non-flat field etc.